pkg/drivers/qemu/qemu.go

```diff
@@ -696,22 +695,21 @@ func (d *Driver) RunQMPCommand(command string) (map[string]interface{}, error) {
 	}
 	jsonCommand, err := json.Marshal(qmpCommand{Command: "qmp_capabilities"})
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "marshal qmp_capabilities")
 	}
 	if _, err := conn.Write(jsonCommand); err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "write qmp_capabilities")
 	}
 	nr, err = conn.Read(buf[:])
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "read qmp_capabilities response")
```

Comment:
knit, but maybe "resp" instead of "response" for shorter wrapping exit message

Reply:
Shortened to "resp"